Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -473,10 +479,29 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         },
         window);
 
+    double last_saved_color_hue = 0;
```

Comment:
Instead of having a variable outside the lambda, you can save the value just before creating the `ColorPicker` and restore it on cancellation.